src/components/Nav/Nav.js

```diff
@@ -1,7 +1,106 @@
 import React from 'react';
+import { css } from 'styled-components';
+import styled from 'styled-components';
```

Comment:
```suggestion
import styled,{ css } from 'styled-components';
```
이렇게 같은 패키지에서 가져온건 이렇게 한줄로 할 수 있어요